presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -44,13 +44,9 @@ Array Functions
 
     Remove duplicate values from the array ``x``.
 
-.. function:: array_dupes(array(bigint)) -> array(bigint)
+.. function:: array_dupes(x) -> array
 
-    Returns an array with elements that are duplicated in the ``array``. Each element is repeated only once and may be in any order.
-
-.. function:: array_dupes(array(varchar)) -> array(varchar)
-
-    Returns an array with elements that are duplicated in the ``array``. Each element is repeated only once and may be in any order.
+    Returns a set of elements that occur more than once in ``x``.
```

Comment:
Let's say that element type of x has to be coercible to bigint or varchar and the return type will be bigint / varchar? Please refer to the documentation of `array_sum`.

Reply:
> Let's say that element type of x has to be coercible to bigint or varchar and the return type will be bigint / varchar? Please refer to the documentation of `array_sum`.

Hmm, why only those two types? It should be applicable to any comparable type right?